### PR TITLE
Pyband: Add msg send

### LINKED ISSUE #2816: Implement MsgSend on Message
Tasks
- [ ] Implement MsgSend data class
- [ ] Implement `as_json` method
- [ ] Implement `get_sender` method
- [ ] Implement `validate` method (Ex. amount is not zero)
- [ ] Add unit test for all method
- [ ] Add documentation

### PR DESCRIPTION
**Pull request by Pitchanai**
Fixed: #2816 

## Implementation details
- Add msg send

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [x] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

File: helpers/pyband/pyband/message.py [hunk 1]
```diff
@@ -59,3 +61,29 @@ def validate(self) -> bool:
...
+            },
+        }
+
+    def get_sender(self) -> Address:
+        return self.from_address
+
+    def validate(self) -> bool:
+        if len(self.amount) == 0:
+            raise ValueError("Expect at least 1 coin")
+
```
Comment (evilpeach) [inline helpers/pyband/pyband/message.py:89]: validate coin?

File: helpers/pyband/tests/message_send_test.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            "band1jrhuqrymzt4mnvgw8cvy3s9zhx3jj0dq30qpte"),
+        from_address=Address.from_acc_bech32(
+            "band1acavyhqpxmz6jt390xze705620q23e4tx4r5he"),
+        amount=[Coin(amount=-1000, denom="uband")]
+    )
+
+    with pytest.raises(
+            ValueError, match="Expect amount more than 0"
+    ):
+        [x.validate() for x in msg_send.amount]
```
Comment (evilpeach) [inline helpers/pyband/tests/message_send_test.py]: It should test like this 
```python
with pytest.raises(ValueError, match="Expect amount more than 0"):
     msg_send.validate()
```

File: helpers/pyband/tests/message_send_test.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                    "denom": "uband"
+                }
+            ]
+        }
+    }
+
+    assert msg_send.get_sender().to_acc_bech32(
+    ) == "band1acavyhqpxmz6jt390xze705620q23e4tx4r5he"
+
+    assert [x.validate() for x in msg_send.amount] == [True]
```
Comment (evilpeach) [inline helpers/pyband/tests/message_send_test.py]: remove

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: helpers/pyband/pyband/message.py [hunk 4]
```diff
@@ -59,3 +61,31 @@ def validate(self) -> bool:
...
+        [x.validate() for x in self.amount]
```
Comment (evilpeach) [inline helpers/pyband/pyband/message.py]:
```
for x in self.amount:
    x.validate()
```

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: helpers/pyband/pyband/message.py [hunk 5]
```diff
@@ -59,3 +61,32 @@ def validate(self) -> bool:
...
+        }
+
+    def get_sender(self) -> Address:
+        return self.from_address
+
+    def validate(self) -> bool:
+        if len(self.amount) == 0:
+            raise ValueError("Expect at least 1 coin")
+
+        for x in self.amount:
```
Comment (taobun) [inline helpers/pyband/pyband/message.py]: for coin in self.amount

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by taobun — APPROVED** [review]

**Pull request merged**